src/fhir-tool-conversion/Exceptions/UnknownFhirTypeException.cs

```diff
@@ -18,6 +18,12 @@ public UnknownFhirTypeException(string message, Type type) : base(message)
             Type = type;
         }
 
+        public UnknownFhirTypeException(Type type, FhirPath path) 
```

Comment:
Make read-only accessible property for Path

Reply:
Added accessor for path